docs/source/markdown/podman-run.1.md

```diff
@@ -946,7 +946,7 @@ For the IPC namespace, the following sysctls are allowed:
 
 Note: if you use the **--ipc=host** option, the above sysctls will not be allowed.
 
-For the network namespace, the following ysctls areallowed:
+For the network namespace, the following sysctls are:
```

Comment:
I could be totally wrong about this. Maybe it should have been `are allowed`

Reply:
yes it should be `are allowed`